src/mumble/licenses.h

```diff
@@ -19,17 +19,11 @@ struct ThirdPartyLicense {
 };
 
 static const char *licenseMumble = 
-	"Copyright (C) 2005-2013, Thorvald Natvig <thorvald@natvig.com>\n"
-	"Copyright (C) 2007, Stefan Gehn <mETz AT gehn DOT net>\n"
-	"Copyright (C) 2007, Sebastian Schlingmann <mit_service@users.sourceforge.net>\n"
-	"Copyright (C) 2007, Trenton Schulz\n"
-	"Copyright (C) 2008-2016, Mikkel Krautz <mikkel@krautz.dk>\n"
-	"Copyright (C) 2008, Andreas Messer <andi@bupfen.de>\n"
-	"Copyright (C) 2008-2016, Stefan Hacker <dd0t@users.sourceforge.net>\n"
-	"Copyright (C) 2008-2011, Snares <snares@users.sourceforge.net>\n"
-	"Copyright (C) 2009-2013, Benjamin Jemlich <pcgod@users.sourceforge.net>\n"
-	"Copyright (C) 2009-2016, Kissaki <kissaki@gmx.de>\n"
-	"Copyright (C) 2010-2016, Jamie Fraser <jamie.f@mumbledog.com>\n"
+	"Copyright (C) 2005-2016 The Mumble Developers\n"
```

Comment:
Seems like we missed to regenerate this file after the license changes. Probably would've been better to have this as a separate commit going in before. Not a big deal though. More interesting would be if we have a way to remind us about running this in the future?